artifacts/definitions/Windows/Timeline/Prefetch.yaml

```diff
@@ -77,6 +77,7 @@ sources:
                     else=TRUE) AND
                 if(condition=dateBefore, then=ExecutionTime < timestamp(string=dateBefore),
                     else=TRUE)
+            GROUP BY ExecutionTime
```

Comment:
Be aware that this group by will cause the entire query to be queued in memory before returning results. Is that something that is desirable? There could be a lot of results.

If flatten() repeats the Execution time it is probably because another column has multiple items do you know why it was producing the wrong output?

Reply:
@scudette yes. The VQL pulls out creation time of the PF file to create another timelined execution event. As with flattern() there were several events, so GROUP BY ExecutionTime is important.

Reply:
WRT the time group. This enables time boxed investigations accross all my artefacts. Default is no entry and should be the standard. Including the option I think is worthwhile for the workflow it provides.